### ISSUE TITLE
EMQX 3.2.3: Node not responding to pings

### ISSUE THREAD

**Issue by gbrehmer** [emqx/emqx#2818]
#### Environment

- OS: Ubuntu 16.04
- Erlang/OTP: 21.3
- EMQ: 3.2.3

#### Description

emqx_ctl Tool can not be used, all requests return "Node not responding to pings". But Hostname is resolvable, all open ports from emq are accessible, and cluster of two nodes is running fine (dashboard, client connection, messaging)

```
root@mqtt-1:~# emqx_ctl cluster status
Node 'emqx@mqtt-1.host' not responding to pings.
root@mqtt-1:~# ping mqtt-1.host
PING mqtt-1.host (127.0.1.1) 56(84) bytes of data.
64 bytes from mqtt-1.host (127.0.1.1): icmp_seq=1 ttl=64 time=0.070 ms
64 bytes from mqtt-1.host (127.0.1.1): icmp_seq=2 ttl=64 time=0.058 ms
64 bytes from mqtt-1.host (127.0.1.1): icmp_seq=3 ttl=64 time=0.064 ms
^C
--- mqtt-1.host ping statistics ---
3 packets transmitted, 3 received, 0% packet loss, time 1998ms
rtt min/avg/max/mdev = 0.058/0.064/0.070/0.005 ms
```

**Comment by biggercousin** [comment]
I also encountered this kind of problem, and I still haven’t solved it in one day.

**Comment by gbrehmer** [comment]
In my case it was my fault (we have a "special" env):
We compiled/packaged the 3.2.3 by ourself and I merged the packaging-part from 3.3 branch into 3.2 branch to get simpler packaging pipeline introduced there. But with 3.3 ekka 0.6 is used, which by default does not use epmd anymore. But we got starting issues with this default setting, so we use env WITH_EPMD=true. We set the variable for our systemd service, but I forgot to set this variable before executing emqx_ctl. Now it's working normally
